pkg/api/admin/service/namespace/service.go

```diff
@@ -22,5 +22,9 @@ func NewService(
 }
 
 func (s Service) ListNamespaces(ctx context.Context) ([]models.Namespace, error) {
```

Comment:
it is pretty simple now but we can create a `unit` tests here. what do you think?

Reply:
yep, I'll do it